staging/src/k8s.io/component-base/featuregate/feature_gate.go

```diff
@@ -296,6 +307,29 @@ func (f *featureGate) Add(features map[Feature]FeatureSpec) error {
 	return nil
 }
 
+func (f *featureGate) OverrideDefault(name Feature, override bool) error {
+	f.lock.Lock()
+	defer f.lock.Unlock()
+
+	known := map[Feature]FeatureSpec{}
+	for name, spec := range f.known.Load().(map[Feature]FeatureSpec) {
+		known[name] = spec
+	}
+
+	if spec, ok := known[name]; !ok {
+		return fmt.Errorf("cannot override default: feature %q is not registered", name)
```

Comment:
if this is a programmer error then we could throw a panic, just like the `Enabled` method does.

Reply:
Taking it a step further, are there any error cases of this method that are _not_ due to programmer error?

Reply:
Actually I take it back. The signature of this method allows for returning an error whereas the signature of the `Enabled` method does not. Let's leave it as it is.

Reply:
SGTM. That would be consistent with the behavior of Add.